web/src/views/Home.vue

```diff
@@ -139,7 +139,7 @@ export default defineComponent({
           .text-2xl.text-brand.font-bold.font-display {{ name }}
           .text-lg.font-bold {{ title }}
           .text-lg {{ email }}
-    .grid.grid-cols-1.gap-4(class="lg:grid-cols-4 lg:gap-32")
+    .grid.grid-cols-1.gap-4(class="lg:grid-cols-3 lg:gap-32")
```

Comment:
Does this look better on the web? Don't have laptop access at the moment. Has phone layout been checked out?

Reply:
This does look a lot better as they are one member fewer this year, without it there is just an empty space when it lists the members. I tried it with firefoxes responsive design and it seems to work.

Reply:
Sounds fair